src/server/index.tsx

```diff
@@ -82,64 +83,81 @@ app.get('/serviceworker.js', (_, res) => {
  */
 app.get('*', async (req, res) => {
   const prefetcher = new PrefetchState();
-
   /** Render first time to register all fetches that are needed for current route */
   const initPrefetch = initJSX(req.path, prefetcher, req.cookies);
-  renderToString(initPrefetch);
+  renderToString(initPrefetch.jsx);
 
   await prefetcher.fetchAll();
 
   /** Initialize and do the actual render which will be sent to the user */
-  const jsx = initJSX(req.path, prefetcher, req.cookies);
+  const { jsx, helmetContext } = initJSX(req.path, prefetcher, req.cookies);
   const reactDom = renderToString(jsx);
-  const HTML = wrapHtml(reactDom, prefetcher);
+  const HTML = wrapHtml(reactDom, prefetcher, helmetContext);
 
   /** Send the finished response to the client */
   res.send(HTML);
 });
 
+interface IInitJSX {
+  jsx: JSX.Element;
+  helmetContext: FilledContext;
+}
+
 /**
  * @summary This is where the magic happens.
  * @description The node server renders all the JSX just as the browser would do it.
  * Worth to remember that 'componentDidMount' will _NOT_ be called in the back-end.
  * Other than that everything is rendered just like in the browser. The StaticRouter uses the
  * requested url to render the corrent corresponding view to the user.
  */
-const initJSX = (location: string, prefetcher: PrefetchState, cookies: { [name: string]: string }): JSX.Element => (
-  <StaticRouter location={location} context={{}}>
-    <Cookies cookies={cookies}>
-      <Prefetched prefetcher={prefetcher}>
-        <ContextWrapper>
-          <App />
-        </ContextWrapper>
-      </Prefetched>
-    </Cookies>
-  </StaticRouter>
-);
+const initJSX = (location: string, prefetcher: PrefetchState, cookies: { [name: string]: string }): IInitJSX => {
+  const helmetContext = {};
+  return {
+    jsx: (
```

Comment:
As with how the other configured tools do it, the context should probably be made outside of this function and be passed as an argument.
The function makes a bit strange by returning two values in this way, as it is just supposed to return JSX.